### DISCUSSION TITLE
Consider adding `--strict` flag

### DISCUSSION THREAD

**Discussion by Pyprohly** [microsoft/pyright#4491] [General]
I frequently like to check short code snippets in both strict and non-strict modes. A command line flag would be faster to use than `# pyright: strict` because it does not involve going away from the console to the file editor to change modes each time.

Why doesn’t Pyright have a `--strict` flag like Mypy?

**Comment by erictraut** [comment]
It's something we could consider adding if there were sufficient interest. I don't think your use case is typical, but I could be wrong.

**Reply by rafrafek** [reply]
I like the way pyright discovers paths, I want to run pyright with default paths and with strict mode enabled. I don't want to specify paths manually for the strict option. 

See https://github.com/encode/uvicorn/pull/1863#issuecomment-1429892092
